recipe/build.sh

```diff
@@ -1,7 +1,36 @@
 #!/usr/bin/env bash
+set -ex
 
-set -exu
+cmake_options=(
+   ${CMAKE_ARGS}
+   "-DCMAKE_BUILD_TYPE=Debug"
+   "-DLAPACK_LIBRARY='lapack;blas'"
+   "-DBUILD_SHARED_LIBS=ON"
+   "-GNinja"
+   ${cmake_mpi_options[@]}
+   ..
+)
 
-cmake $CMAKE_ARGS -GNinja -B_build
-cmake --build _build
-cmake --install _build
+mkdir -p _build
+pushd _build
+
+FFLAGS="-fno-backtrace" cmake "${cmake_options[@]}"
```

Comment:
Don't override the `FFLAGS` set by the environment but append new flags.

Reply:
What about now?